test/test_field.jl

```diff
@@ -113,7 +113,7 @@ function run_field_interpolation_tests(FT, arch)
 
     # Maximum expected rounding error is the unit in last place of the maximum value
     # of f over the domain of the grid.
-    ε_max = f.(nodes(grid, (Face(), Face(), Face()), reshape=true)...) |> maximum |> eps
+    ε_max = (f.(nodes(grid, (Face(), Face(), Face()), reshape=true)...) |> maximum |> eps) * 10

```

Comment:
```suggestion
    xf, yf, zf = nodes(grid, (Face(), Face(), Face()), reshape=true)
    f_max = maximum(f.(x, y, z))
    ε_max = eps(f_max)
    tolerance = 10 * ε_max
```

Let's make an effort...

Reply:
I accidentally made this change on the wrong branch but will change it for #2912